src/resolvers/Chat/creator.ts

```
@@ -1,12 +1,23 @@
-import { User } from "../../models";
-import type { ChatResolvers } from "../../types/generatedGraphQLTypes";
+import { InterfaceUser, User } from "../../models";
```

ESLint

[error] Imports "InterfaceUser" are only used as type.

(@typescript-eslint/consistent-type-imports)